network/dag/publisher.go

```diff
@@ -123,9 +131,24 @@ func (s *replayingDAGPublisher) publish(ctx context.Context) {
 	}
 }
 
+func (s *replayingDAGPublisher) handlePrivateTransaction(tx Transaction) error {
+	body, err := json.Marshal(tx)
```

Comment:
you could use `tx.Data()`, which contains the original TX data as byte slice (and does not return an error)